src/components/themeToggleBtn.jsx

```diff
@@ -0,0 +1,27 @@
+import { ThemeContext } from '@/store/theme-context'
+import Image from 'next/image'
+import { useContext } from 'react'
+
+export default function ThemeToggleButton() {
+  const { theme, toggleTheme } = useContext(ThemeContext)
```

Comment:
And then you can rename `toggleTheme` -> `toggle`, I guess it will be used only in one place. So `toggle` is a good name and it won't conflict with the component name `ToggleTheme`